test/unit/pages/ClinicPatients.test.js

```diff
@@ -149,6 +149,10 @@ describe('ClinicPatients', () => {
         settingClinicPatientLastReviewed: defaultWorkingState,
         revertingClinicPatientLastReviewed: defaultWorkingState,
       },
+      patientListFilters: {
+        patientListSearchTextInput: '',
+        isPatientListVisible: true
```

Comment:
The variable names are quite long here but I slightly prefer greppability over readability when dealing with redux because the variables end up getting referenced everywhere.